TestClass.cpp

```diff
@@ -0,0 +1,234 @@
+#include <iostream>
+#include <cmath>
+#include <cassert>  
+
+using namespace std; //use std::cout
+
+#define FILE (__FILE__)
+#define LINE (__LINE__)
+
+// #define LOG_LENGTH (300)
+const int LOG_LENGTH = 300;
+
+void print_log(const char *filename,
+               int         line,
+               const char *message)
+{
+    printf("F:%s L:%d : %s\n", filename, line, message);
+}
+
+class TestClass
+{
+    private:
+        int var1;
+        int var2;
+        // double var3;
+        // double var4;
+        int max_average_value;
+        int max_square_root_value;
+
+    public:
+    //get and set functions are public
+
+        TestClass(int a, int b)
```

Comment:
a and b are not being changed inside this function, so you can change the param types for this function

Reply:
done